--- modsecurity-apache_2.5.9/rules/modsecurity_crs_10_config.conf~	2008-04-24 18:48:08.000000000 +0200
+++ modsecurity-apache_2.5.9/rules/modsecurity_crs_10_config.conf	2009-08-11 14:54:14.403528140 +0200
@@ -110,7 +110,7 @@
 #      "Apache/2.2.0 (Fedora)"   
 #      For this directive to work, you need to set Apache ServerTokens
 #      to Full (this is the default option)
-SecServerSignature "Apache/2.2.0 (Fedora)"
+SecServerSignature "Apache/2.2.0 (PLD)"
 
 # Add ruleset identity to the logs
 #
